odh/overlays/moc/monitoring/servicemonitors/cluster-metrics-servicemonitor.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: cluster-monitor-federation
+  labels:
+    monitor-component: cluster-resources
+spec:
+  endpoints:
+    - bearerTokenSecret:
+        key: token
+        name: prometheus-k8s-token-g9n8s
```

Comment:
This token secret is unique for each SA "instance" (hence the `-g8n8s` suffix). If we would redeploy the Service Account, this token secret name would change. You'd probably need to create a new SA secret with a static name and refer to that one.

https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#to-create-additional-api-tokens

```
apiVersion: v1
kind: Secret
metadata:
  name: a-new-static-name
  annotations:
    kubernetes.io/service-account.name: prometheus-k8s
type: kubernetes.io/service-account-token
```

Reply:
@tumido I have added a secret in the secrets dir can you give it a look?